e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/submit-deal-cancellation-in-future.spec.js

```diff
@@ -0,0 +1,113 @@
+import { TFM_DEAL_STAGE, TFM_FACILITY_STAGE } from '@ukef/dtfs2-common';
+import relative from '../../relativeURL';
+import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1 } from '../../../../../e2e-fixtures';
+import caseDealPage from '../../pages/caseDealPage';
+import { tomorrow } from '../../../../../e2e-fixtures/dateConstants';
+import checkDetailsPage from '../../pages/deal-cancellation/check-details';
+import dealsPage from '../../pages/dealsPage';
+import { successBanner } from '../../partials';
+import facilitiesPage from '../../pages/facilitiesPage';
+
+context('Deal cancellation - effective from date in past', () => {
```

Comment:
The file name and this description is confusing me:

```
- submit-deal-cancellation-in-future.spec.js
- 'Deal cancellation - effective from date in past'
```

Should it be e.g

```
- submit-deal-cancellation-X-date-in-future.spec.js
- 'Deal cancellation - X date in the future'
```

Reply:
Have renamed, let me know if that's clearer